### PR TITLE
exclude federalist.json and pages.json from deployments

### PR DESCRIPTION
**Pull request by davemcorwin**
## Changes proposed in this pull request:
- Also exclude federalist.json and pages.json from deployments

## security considerations
None

### CODE REVIEW

File: src/build.py [hunk 1]
```diff
@@ -110,7 +110,12 @@ def run_step(returncode, msg):
                 CLONE_DIR_PATH,
                 dict(
                     headers=dict([('cache-control', cache_control)]),
-                    excludePaths=['*/Dockerfile', '*/docker-compose.yml'],
+                    excludePaths=[
+                        '*/Dockerfile',
+                        '*/docker-compose.yml',
+                        '*/federalist.json',
+                        '*/pages.json'
```
Comment (hursey013) [inline src/build.py:117]: [question] I wonder about this one just because it seems possible that folks may have a `pages.json` somewhere within their site's structure that is not related to cloud.gov Pages. Should these be scoped to the root?

**Review by hursey013 — APPROVED** [review]

File: src/build.py (hunk 1 above)
Comment (davemcorwin) [reply src/build.py:117]: we can start with that
